src/test/java/seedu/jarvis/model/planner/DeadlineTest.java

```diff
@@ -0,0 +1,49 @@
+package seedu.jarvis.model.planner;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.util.Date;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.jarvis.model.address.tag.Tag;
+
+
+
+class DeadlineTest {
+
+    @Test
+    void addPriority() {
```

Comment:
Maybe we could standardise all test method names in the format: ```write_emptyInput_exceptionThrown()``` to make clearer the purpose of the test.